librz/include/rz_cmd.h

```diff
@@ -431,7 +431,7 @@ typedef struct rz_core_plugin_t {
 	RzCmdCb fini;
 } RzCorePlugin;
 
-typedef bool (*RzCmdForeachNameCb)(RzCmd *cmd, const char *name, void *user);
+typedef bool (*RzCmdForeachNameCb)(RzCmd *cmd, RzCmdDesc* desc, const char *name, void *user);
```

Comment:
```suggestion
typedef bool (*RzCmdForeachNameCb)(RzCmd *cmd, const RzCmdDesc* desc, const char *name, void *user);
```
So we can be sure callers won't change desc.